generator/world_map_generator.hpp

```diff
@@ -53,8 +53,27 @@ class WorldMapGenerator
     // This functor is called by m_merger after merging linear features.
     void operator()(feature::FeatureBuilder const & fb) override
     {
-      // Skip small ways. This check is enough, because classifier types check was made in m_typesCorrector.
-      if (scales::IsGoodForLevel(scales::GetUpperWorldScale(), fb.GetLimitRect()))
+      static const uint32_t ferryType = classif().GetTypeByPath({"route", "ferry"});
```

Comment:
constexpr

Reply:
GetTypeByPath() is not constexpr

Reply:
Right! It would be great to make these expressions constexpr in the future...